pages/blog/release-notes-3.0.0.md

```diff
@@ -0,0 +1,329 @@
+- added `title`, `summary`, and `externalDocs` fields in Server Object
+- added `title`, and `summary` fields in Channel Object
+- added `title` field in Operation Object and Operation Trait Object
+
+```
+asyncapi: 3.0.0
+servers:
+  SomeServer:
+    title: Some Server title
+    summary: This some server is for something
+    externalDocs:
+      ...
+channels:
+  SomeChannel:
+    title: Some channel title
+    summary: Some channel summary
+operations:
+  SomeOperation:
+    title: Some operation title
+    traits:
+      - title: Some operation traits title
+```
+
+Issues: [#795](https://github.com/asyncapi/spec/issues/795) | Pull request: [#796](https://github.com/asyncapi/spec/pull/796)
+
+### Cleaning up the root object
+There was two meta information lingering in the root of the AsyncAPI object, which did not make much sense since we have the `info` object for all the meta information.
+
+Therefore the root `tags` and `externalDocs` have been moved to the info object.
+
+```
+asyncapi: 3.0.0
+info:
+  ...
+  externalDocs:
+    description: Find more info here
+    url: https://www.asyncapi.org
+  tags:
+    - name: e-commerce
+...
+```
+
+Pull request: [#794](https://github.com/asyncapi/spec/pull/794)
+
+### Splitting out server URL into host and pathname
+There has been some confusion about what the `url` of a server should contain, is it both protocol + host + path? What about the protocol field then? Therefore each field now has its own field for the host, path, and protocol.
+
+```
+asyncapi: 3.0.0
+servers:
+  localhost:
+    host: localhost
+    path: /api/v1,
+    protocol: mqtt
+```
+
+Issues: [#547](https://github.com/asyncapi/spec/issues/547), [#274](https://github.com/asyncapi/spec/issues/274) | Pull request: [#888](https://github.com/asyncapi/spec/pull/888)
+
+### More reusable objects in components
+This is a bit of a mixture between some of the features, that all added a little to this. It's now possible to add more stuff under components:
+- Replies
+- Reply addresses
+- Tags
+- External docs
+- Operations
+- Channels
+
+```
+asyncapi: 3.0.0
+components:
+  ...
+  replies:
+    ...
+  replyAddresses:
+    ...
+  tags: 
+    ...
+  externalDocs:
+    ...
+  operations:
+    ...
+  channels:
+    ...
+```
+
+Issues: [#829](https://github.com/asyncapi/spec/issues/829) | Pull request: [#847](https://github.com/asyncapi/spec/pull/847), [#792](https://github.com/asyncapi/spec/pull/792), [#806](https://github.com/asyncapi/spec/pull/806), [#827](https://github.com/asyncapi/spec/pull/827)
+
+### New trait behavior
+Traits in v2 always replaced any duplicate properties that were defined both in traits and the associated object. This meant for example if the message traits defined headers and the message object did as well, only the message trait headers would be applied because it overwrote anything you wrote in the message object.
```

Comment:
```suggestion
Traits in v2 always replaced any duplicate properties that were defined both in traits and the associated object. This meant, for example, if the message traits defined headers and the message object did as well, only the message trait headers would be applied because it overwrote anything you wrote in the message object.
```